ChangeLog.md

```diff
@@ -3,10 +3,17 @@
 
 ## [(diff)](https://github.com/haskell-nix/hnix/compare/0.16.0...0.17.0#files_bucket) 0.17.0
 
+* Introduction:
+  * `Nix.Value`
+    * [(link)](https://github.com/haskell-nix/hnix/pull/1046/files) Constraint `NVConstraint f = (Comonad f, Applicative f)` was introduced, in order to unify builder `mkNV*` and `NV*` patterns.
+
```

Comment:
Would note that Additional & Breaking are generally enough.

For example, this change does not belong into "Introduction", it can be thought to belong to Additional, but it changes type signatures in a number of places - so it is a Breaking change.

The first most important client type of ChangeLogs in Haskell projects - is downstream developers. & generally under semantic agenda of supporting the update, people look into changelogs. So the most important and clear thing ChangeLog should do - is to have as many breaking changes described precisely as possible. That it is an introduction of something into the code - can be said in explanation of breaking changes.

Reply:
I was originally put it into breaking, but it comes to me that it is jut short hands for `Comonad f, Applicative f`.
An introduction of short hand for the new constraint requirements, I think it is both introduction and breaking, sort of dangling in the middle. 

Since you remind me of the change logs focus is on breaking changes and reading experience of developers , move back to breaking does seem to make more sense.  Thanks, surely will modify it.

Reply:
done in #1048